### ISSUE TITLE
Updating Peer Dependencies

### ISSUE THREAD

**Issue by caraya** [wordpress-clients/wp-api-angular#41]
I'm trying to work with wp-api-angular with Angular 6. However, the library requires a peer of version 4 which means it won't work with my current setup. 

Is there a new version of the client? Beta8 was released almost a year ago. If there isn't one, how safe is it to update the peer dependencies of the client?

**Comment by ritsrivastava01** [comment]
any update..

**Comment by jokin** [comment]
Doesn't seem that they are updating the repo, I'm gona fork it and change the package.json dependencies to try, if it work I will send a PR.

**Comment by jokin** [comment]
well, you can use 

> "wp-api-angular": "jokin/wp-api-angular#bec320",

or fork and use your own repo to get rid of the peer dependencies warnings.

**Comment by s4m0r4m4** [comment]
@jokin did you end up making a new fork successfully? I'm looking to update this repo and didn't want to redo work

**Comment by jokin** [comment]
> @jokin did you end up making a new fork successfully? I'm looking to update this repo and didn't want to redo work

No, I abandoned the idea of doing what I was thinking of with angular so I don't recall if I even tried to change the package.json

**Comment by s4m0r4m4** [comment]
I bumped the versions to Angular v11 and should have fixed the issues in this PR: https://github.com/wordpress-clients/wp-api-angular/pull/48
